js/leaflet-sidebar.js

```diff
@@ -13,6 +18,12 @@ L.Control.Sidebar = L.Control.extend({
         if (L.Browser.touch)
             L.DomUtil.addClass(this._sidebar, 'leaflet-touch');
 
+        // Create close button and attach it if configured
+        if (this.options.closeButton === true) {
+            this._closeButton = L.DomUtil.create('a', 'sidebar-close-button u-display-none', this._sidebar);
+            this._closeButton.innerHTML = '&times;';
+        }
```

Comment:
You should also make sure you set this to the tabindex = 0 so that it is accessible. 

this._closeButton.tabIndex = 0;